commands/exportcommands.py

```
@@ -0,0 +1,1002 @@
+            print(f"   ⚠️ Could not fetch existing version history: {e}")
+        return None
+    
+    def _increment_version(self, version):
+        """Increment minor version for new export (patch for small changes)"""
+        try:
+            major, minor, patch = map(int, version.split('.'))
+            # Increment minor version for this update
+            new_version = f"{major}.{minor + 1}.0"
+            print(f"   📋 Incremented version: {version} → {new_version}")
+            return new_version
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)